doc/develop/getting_started/index.rst

```diff
@@ -317,12 +317,12 @@ additional Python dependencies in a `Python virtual environment`_.
 
             west zephyr-export
 
-      #. Zephyr's ``scripts/requirements.txt`` file declares additional Python
-         dependencies. Install them with ``pip``.
+      #. Zephyr's ``packages`` helper command can be used to install Python
+         dependencies.
```

Comment:
"packages" is quite vague. There are two, more specific words already in use: "requirements" and "dependencies". Can we stick to these two and not add another, vaguer third terminology? Is there anything wrong with the existing terms?


We're already struggling with the vague "projects" and "modules"... I digress sorry.

BTW, what categories do we have now and which ones is this intended for? Build dependencies, debug dependencies, code running on device, other?

Reply:
I'm not convinced that "requirement" or "dependencies" would be better. The intention is to make it easier for users and module owners to list or install packages from different package managers.

If we look at the [python documentation](https://packaging.python.org/en/latest/tutorials/installing-packages/):

> It’s important to note that the term “package” in this context is being used to describe a bundle of software to be installed.

If we were to add other managers; the terminology is very similar, for example:

- `cargo` is the [Rust](https://www.rust-lang.org/) [package manager](https://doc.rust-lang.org/cargo/appendix/glossary.html#package-manager). Cargo downloads your Rust [package](https://doc.rust-lang.org/cargo/appendix/glossary.html#package)’s dependencies, compiles your packages, makes distributable packages, and uploads them to [crates.io](https://crates.io/), the Rust community’s [package registry](https://doc.rust-lang.org/cargo/appendix/glossary.html#package-registry).
- `npm` ([Node Package Manager](https://docs.npmjs.com/about-npm))
- `apt` ([Advanced Packaging Tool](https://ubuntu.com/server/docs/package-management))
- `dnf` is the [Fedora Project package manager](https://docs.fedoraproject.org/en-US/quick-docs/dnf/)
- `pacman` The [pacman](https://pacman.archlinux.page/) [package manager](https://en.wikipedia.org/wiki/Package_manager) is one of the major distinguishing features of [Arch Linux](https://wiki.archlinux.org/title/Arch_Linux).
- `choco` from [chocolatey](https://chocolatey.org/) The Package Manager for Windows
- `brew` from [Homebrew](https://brew.sh/) The Missing Package Manager for macOS (or Linux)

Reply:
`packages` actually is already long IMO, some alternatives:

- `pkg`/`pkgs`
same, but shorter
- `pacman`
maybe too confusing with the arch package manager
- `zpm`
"Zephyr package manager", I don't think many would favor this one

Reply:
Yeah don't pick pacman, it might have originated from arch but it's used in other things and even on windows.

Reply:
I like `pkgs` 🤷

Reply:
had a look through PR properly, packages is fine to me

Reply:
i'm fine with `packages`.

But I noticed the part: `Zephyr's ``packages`` helper command`.
Perhaps better to use the term `west extension command`, for example:
`The Zephyr west extension command, ``west packages``, can be used to ...`

Reply:
Updated